mayan/apps/reviewers/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from mayan.apps.databases.model_mixins import ExtraDataModelMixin
+from mptt.models import MPTTModel
+
+class Dashboard(ExtraDataModelMixin, MPTTModel):
+    pass
+
+class Candidate(ExtraDataModelMixin, MPTTModel):
```

Comment:
We may want to add more data validation such as regex, but for now this is good. I'll add a task so we can follow up on this.

Reply:
Created issue #33 to close the loop on this.